gdcdatamodel/gdc_postgres_admin.py

```diff
@@ -237,40 +237,6 @@ def kill_blocking_psql_backend_processes(engine):
         execute(engine, sql_cmd)
 
 
-def create_tables_force(engine, delay, retries, namespace=None):
-    """Create the tables and **KILL ANY BLOCKING PROCESSES**.
-
-    This command will spawn a process to create the new tables in
-    order to find out which process is blocking us.  If we didn't do
-    this concurrently, then the table creation will have disappeared
-    by the time we tried to find its blocker in the postgres backend
-    tables.
-
-    """
-
-    logger.info('Running table creator named %s', app_name)
-    logger.warning('Running with force=True option %s', app_name)
-
-    from multiprocessing import Process
-    p = Process(target=create_graph_tables, args=(engine, delay, namespace))
-    p.start()
-    time.sleep(delay)
-
-    if p.is_alive():
-        logger.warning('Table creation blocked!')
-        kill_blocking_psql_backend_processes(engine)
```

Comment:
We should also remove the code that used to support the "force" option (`kill_blocking_psql_backend_processes`, `lookup_blocking_psql_backend_processes`, `is_blocked_by_no_kill`, `BLOCKING_SQL`, etc.). It might take a couple rounds of removing things and then checking for code that isn't used anymore. I wouldn't be surprised if some imports go away as well.